### DISCUSSION TITLE
Maintenance release of Tablesaw

### DISCUSSION THREAD

**Discussion by ccleva** [jtablesaw/tablesaw#1261] [General]
Hi all,

As discussed in [another thread](https://github.com/jtablesaw/tablesaw/discussions/1233), I made a maintenance release of the project. It is based on the 0.43.1 version and includes all fixes and package upgrades from this repository.

The maintenance fork is here https://github.com/tlabs-data/tablesaw.

A release candidate [0.43.2-rc1](https://github.com/tlabs-data/tablesaw/releases/tag/v0.43.2-rc1) is available in Maven Central under the [tlabs-data organization](https://central.sonatype.com/search?q=tlabs-data&smo=true&sort=published).

Please report issues you might have with the release candidate in the [forked repository](https://github.com/tlabs-data/tablesaw/issues), and feel free to report it works in the [discussion](https://github.com/tlabs-data/tablesaw/discussions/19).

I will release version 0.43.2 once it is confirmed there is no blocking issues. Additional releases will follow with more fixes, but for the moment no new feature will be implemented.

Enjoy!

**Comment by ccleva** [comment]
No issues were reported on the release candidate, the release [0.43.2](https://github.com/tlabs-data/tablesaw/releases/tag/v0.43.2) is now available in Maven Central under the [tlabs-data organization](https://central.sonatype.com/search?q=tlabs-data&smo=true&sort=published).

**Comment by ccleva** [comment]
A new maintenance release ([v0.43.3](https://github.com/tlabs-data/tablesaw/discussions/33)) is available in [Maven Central](https://central.sonatype.com/search?q=tlabs-data&smo=true&sort=published).

The new release contains additional fixes from this repository that needed a bit more testing, and new code to fix some issues reported here. Feel free to [provide feedback](https://github.com/tlabs-data/tablesaw/discussions) or [raise issues](https://github.com/tlabs-data/tablesaw/issues) in the forked repository.

**Comment by ccleva** [comment]
A new maintenance release ([v0.44.0](https://github.com/tlabs-data/tablesaw/discussions/49)) is available in [Maven Central](https://central.sonatype.com/search?q=tlabs-data&smo=true&sort=published).

The new release focuses mainly on deprecated code removal, which required some internal refactoring of the `jsplot` module, and code improvement for maintainability. No API change besides deprecated code removal. Includes a few fixes for issues discovered during cleanup. Feel free to [provide feedback](https://github.com/tlabs-data/tablesaw/discussions) or [raise issues](https://github.com/tlabs-data/tablesaw/issues) in the forked repository.

Note that shaded jars are no longer produced.

**Comment by benmccann** [comment]
I don't consider Tablesaw to be unmaintained and would prefer not to fork it. If there are changes that need to make it into this repo, let's work together to figure that out

**Reply by benmccann** [reply]
@ccleva is there something we can do to prevent folks from thinking https://github.com/tlabs-data/tablesaw is the place to go now that we've gotten a lot of your PRs in here? Maybe delete the repo, post a notice on the readme and archive it, or edit the description in the sidebar?

**Reply by ccleva** [reply]
@benmccann I removed the releases from the sidebar and the discussion was already updated to point back here for new versions. As some artifacts were deployed to maven central the repository needs to stay.
